modules/mod_ginger_collection/templates/collection/prevnext.tpl

```diff
@@ -0,0 +1,19 @@
+{% if q.query_id and q.current %}
```

Comment:
This conditional covers the case where you land directly on a collection object page and no query context is available, right?

Reply:
Yes